ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/ssh/SshServiceClientImpl.java

```diff
@@ -29,6 +31,7 @@
  *
  * @author Sergii Leschenko
  */
+@Singleton
```

Comment:
Why did you add this annotation?
It's already bound as singleton in CoreGinModule.